jenkins.bash

```diff
@@ -98,6 +106,16 @@ EOF
 	coverage combine "--rcfile=${COVERAGE_RC}" $(find . -name '.coverage.*')
 	coverage xml "--rcfile=${COVERAGE_RC}"
 	codecov --file coverage.xml
+
+	if [ -d conformance ]
+	then
+		pushd conformance
+		git add --all
+		git diff-index --quiet HEAD || git commit -m "CWL Jenkins Build bot"
```

Comment:
Can we add more information to the commit?

Reply:
Done! Now the commit message includes:

- cwltool version
- CWL version
- commit hash
- Python version
- Container (docker or singularity)